tests/php/test-class-amp-nav-menu-dropdown-sanitizer.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Tests for AMP_Nav_Menu_Dropdown_Sanitizer.
+ *
+ * @package AMP
+ */
+
+use Amp\AmpWP\Tests\PrivateAccess;
+
+/**
+ * Class Test_AMP_Nav_Menu_Dropdown_Sanitizer
+ *
+ * @covers AMP_Nav_Menu_Dropdown_Sanitizer
+ */
+class Test_AMP_Nav_Menu_Dropdown_Sanitizer extends WP_UnitTestCase {
+
+	use PrivateAccess;
+
+	/**
+	 * Test add_buffering_hooks.
+	 *
+	 * @covers AMP_Nav_Menu_Dropdown_Sanitizer::add_buffering_hooks
+	 */
+	public function test_add_buffering_hooks() {
+		global $wp_filter;
+
+		$filter = 'walker_nav_menu_start_el';
+		remove_all_filters( $filter );
+
+		// The filter should not be added if certain args are not passed.
+		AMP_Nav_Menu_Dropdown_Sanitizer::add_buffering_hooks( [] );
+		$this->assertEquals( false, isset( $wp_filter[ $filter ] ) );
```

Comment:
As opposed to checking the global variable with `isset( $wp_filter[ $filter ] )` can't `has_filter( $filter )` be used?

```suggestion
		$this->assertEquals( false, has_filter( $filter ) ) );
```